### ISSUE TITLE
Voiceover accessibility not great

### ISSUE THREAD

**Issue by deathdogg** [CodeHubApp/CodeHub#563]
I use GitHub both on the command line and on the website, but I downloaded this app to carry around my repos. I am a blind coder and as of right now, Apple's Voiceover screenreader reads most elements but there are a lot of unlabeled buttons throughout the app.

**Comment by sirisak1985** [comment]
g

**Comment by lawrenceper** [comment]
I agree. I downloaded this app for the same reason. Some buttons are not labeled for users who use iOS VoiceOver. Can you please fix?

**Comment by sirisak1985** [comment]
issue

**Comment by sirisak1985** [comment]
issue
